reflex/components/radix/themes/layout/list.py

```diff
@@ -42,29 +42,32 @@ class BaseList(Component):
     tag = "ul"
 
     # The style of the list. Default to "none".
-    list_style_type: Var[
+    list_style_type: Optional[
         Union[LiteralListStyleTypeUnordered, LiteralListStyleTypeOrdered]
-    ]
+    ] = "none"
+
+    # A list of items to add to the list.
+    items: Optional[Union[Iterable, Var[Iterable]]] = None
```

Comment:
```suggestion
    items: Var[Iterable]
```

this one can just be `Var`, since it's handled in a ForEach in that case